server/routes/newSeats.js

```diff
@@ -0,0 +1,76 @@
+// Get dependencies
+const express = require('express');
+const router = express.Router();
+const mongodb = require('mongodb');
+const post = require()
+
+router.get('/', (req, res) => {
+res.send([]);
+    // grab members from cses page
+    var MongoClient = mongodb.MongoClient;
+
+    var newSeats = new mongoose.Schema({
+      //var  type
+      room: Number,
+      seatNumber: Number
+  });
+
+    //mongoose schema into a mongoose model
+    var Seats = mongoose.model("Seats", newSeats)
+
+    MongoClient.connect(DB_URL,function(err,db){
+        // Let the front end component handle any errors
+        if (err) {
+            res.send(err);
+            return;
+        }
+
+        //Grab all the members info and format them
+        var collection = db.collection('newSeats');
+        console.log("here");
+
+        //5. Is finding records necessary or is there something else I should use?
+        collection.find({}).toArray(function(err,newSeats){
```

Comment:
We don't need to be finding any records for this route, we just need to be saving the body of the post request after validating it with a secret key